systemvm/patches/debian/config/opt/cloud/bin/set_redundant.py

```diff
@@ -1,47 +0,0 @@
-#!/usr/bin/python
```

Comment:
True, I haven't looked for references from the root of the project, my bad...

Unfortunately that mean the [test/systemvm](/apache/cloudstack/tree/8a2391336c305a0209b70062ccdd41bbe0bdff44/test/systemvm) are underused, because the method `set_cl` has been removed a while ago by bdda01d2695ef4426857a85ec435cefcab3a0362.